liquigraph-core/src/test/java/org/liquigraph/core/io/GraphJdbcConnectorTest.java

```diff
@@ -17,7 +17,7 @@
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
-    private GraphJdbcConnector connector = new GraphJdbcConnector();
+    private GraphJdbcConnector connector = new GraphJdbcConnector(new LockGuardian());
```

Comment:
If you choose to keep the injection explicit in the test, shouldn't a mock be injected instead of the real `LockGuardian`? You'll create as many shutdown hooks as you have tests, and you may just want to verify the locking is done, not actually do it.